package.json

```diff
@@ -25,7 +25,7 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "@playwright/test": "^1.22.2",
+    "@playwright/test": "^1.23.0",
```

Comment:
Can use the Snyk bot to update dependencies frequently or use Dependabot, whatever works. Probably better to just use one of them to keep it up to date?